scripts/pylint_extensions_test.py

```diff
@@ -189,3 +189,44 @@ def test_finds_import_from(self):
         ):
             checker_test_object.checker.visit_importfrom(
                 importfrom_node2)
+
+
+class BackslashContinuationCheckerTests(unittest.TestCase):
+
+    def test_finds_backslash_continuation(self):
+        checker_test_object = testutils.CheckerTestCase()
+        checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.BackslashContinuationChecker)
+        checker_test_object.setup_method()
+        node = astroid.scoped_nodes.Module(name='test', doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+        # pylint: disable=backslash-continuation
```

Comment:
Do you need to disable the rule for this complete block? I think disabling for a single line would be suffice.

Reply:
Yes, fixed this. PTAL!